orphanage_APP/resources/views/layouts/app.blade.php

```diff
@@ -92,5 +98,12 @@
             @yield('content')
         </main>
     </div>
+ <script src="{{ asset('js/jquery-3.5.1.min.js') }}"></script>
+ <script>
+    $("#menu-toggle").click(function(e) {
+      e.preventDefault();
+      $("#wrapper").toggleClass("toggled");
+    });
+  </script>
```

Comment:
I cannot see where this script works across all files?

Reply:
The script currently is not used anywhere in the code but it will be used in future migration of our app navbar into sidenav. I think for now if it is okay i will comment it out only